### PR TITLE
Improvements to Yii1

### PR DESCRIPTION
**Pull request by DavertMik**
Yii1 connector improved

* AR metadata cache is cleaned
* db connection closed
* `regenerateId` of CHttpSession disabled

### CODE REVIEW

File: src/Codeception/Lib/Connector/Yii1.php [hunk 1]
```diff
@@ -86,17 +86,30 @@ public function doRequest($request)
...
+
+        if ($app->hasComponent('session')) { // disable regenerate id in session
+            $app->setComponent('session', Stub::make('CHttpSession', ['regenerateID' => false]));
+        }
+
+        $app->onEndRequest->add([$this, 'setHeaders']);
+        $app->run();
+
+        // close connection
+        $app->getDb()->setActive(false);
```
Comment (tomtomsen) [inline src/Codeception/Lib/Connector/Yii1.php]: why didn't you check if db component exists?
```php
if ($app->hasComponent('db'))
```

Comment (DavertMik) [inline src/Codeception/Lib/Connector/Yii1.php]: thanks

File: src/Codeception/Lib/Connector/Yii1.php [hunk 2]
```diff
@@ -86,17 +86,32 @@ public function doRequest($request)
...
+        }
+
+        $app->onEndRequest->add([$this, 'setHeaders']);
+        $app->run();
+
+        if ($app->hasComponent('db')) {
+            // close connection
+            $app->getDb()->setActive(false);
+            // cleanup metadata cache
+            $property = new \ReflectionProperty(\CActiveRecord::class, '_md');
```
Comment (samdark) [inline src/Codeception/Lib/Connector/Yii1.php]: `::class` is PHP 5.5+ while Codeception requirement is PHP 5.4.

Comment (DavertMik) [inline src/Codeception/Lib/Connector/Yii1.php]: Ups, thanks

**Pull request merged**